src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowPublishApplication.java

```diff
@@ -62,14 +62,14 @@ public class ElectricFlowPublishApplication extends Recorder implements SimpleBu
 
   // ~ Static fields/initializers ---------------------------------------------
 
-  public static final String deploymentPackageName = "deployment_package.zip";
-  private static final Log log = LogFactory.getLog(ElectricFlowPublishApplication.class);
-  private static List<File> zipFiles = new ArrayList<>();
+  public final static String deploymentPackageName = "deployment_package.zip";
```

Comment:
What reason is there for this change? [Java Language specification, § 8.1.1, 8.3.1, 8.4.3](https://docs.oracle.com/javase/specs/jls/se16/preview/specs/sealed-classes-jls.html) and [9.4](https://docs.oracle.com/javase/specs/jls/se11/html/jls-9.html) recommends the original order. I can't imagine SpotBugs complaining about something recommended in the JLS.